cfpb/cfgov-refresh
# Fixes to housing counselor semantics

## Changes

- Adds URL-encoding to text in Facebook sharing link URL to handle spaces (`Find a housing counselor` to `Find%20a%20housing%20counselor`).
- Removes extraneous spaces in email address link.
- Convert `View Paperwork Reduction Act statement` link that doesn't go to a page to a button link.

## Testing

1. pull down branch and visit /find-a-housing-counselor/?zipcode=05201&s3=True
2. Scroll to bottom and see `View Paperwork Reduction Act statement`.

## Screenshots

Before
<img width="322" alt="screen shot 2017-07-20 at 12 58 31 pm" src="https://user-images.githubusercontent.com/704760/28429640-5d58914e-6d4b-11e7-8e3f-4fe636c03cf9.png">

After
<img width="373" alt="screen shot 2017-07-20 at 12 58 40 pm" src="https://user-images.githubusercontent.com/704760/28429641-5d599c7e-6d4b-11e7-84b2-f9ebf85771c7.png">

## Notes

- I think the link styles before were incorrect, because now they're our link button styles from CFv4.
